python/pytests/pandas_source_test.py

```diff
@@ -1,37 +1,41 @@
-import random
-
 import kaskada as kd
 import pandas as pd
 
+def test_float_microseconds(golden) -> None:
+    data = {
+        'time': [1671477472.026119],
```

Comment:
seems like this is actually a seconds timestamp.  if it was microseconds it would be `1671477472026119.123`